src/gui/log/logmodel.cpp

```diff
@@ -172,7 +172,7 @@ void LogPeerModel::handleNewMessage(const Log::Peer &peer)
 {
     const QString time = QDateTime::fromMSecsSinceEpoch(peer.timestamp).toString(Qt::SystemLocaleShortDate);
     const QString message = peer.blocked
-            ? tr("%1 was blocked due to %2", "0.0.0.0 was blocked due to reason").arg(peer.ip, peer.reason)
+            ? tr("%1 was blocked %2", "0.0.0.0 was blocked due to reason").arg(peer.ip, peer.reason)
```

Comment:
It doesn't look like this format can produce a valid message...

Reply:
Or do you want to say that libtorrent reason string is started with "due to"?

Reply:
@glassez

As far as I can tell, we build these strings ourselves and they are already prefixed with "due to", "because of" and such:

https://github.com/qbittorrent/qBittorrent/blob/defdd5121c940dfc3b807401338e15f66adfd671/src/base/bittorrent/session.cpp#L4477

As a sanity check, xavier2k6 has confirmed this fixes the problem: https://github.com/qbittorrent/qBittorrent/issues/13352#issuecomment-690105765

Reply:
@glassez 
It was unexpectedly modified in this commit: https://github.com/qbittorrent/qBittorrent/commit/fd89717330b826494ae37b48ddcdbada11c0b9a7#diff-48bde9140be4ac1ca5199de1b9656f9cR183

Reply:
It was modified based on the belief that the lower level returns strings in the correct format. But, unfortunately, this turned out to be untrue:
>we build these strings ourselves and they are already prefixed with "due to", "because of" and such

The root of the problem is not here. So if you're not too lazy, you'd rather fix it where it grows from.

Reply:
@glassez 

Sorry, it is not clear to me what you are suggesting. The root of the problem was indeed this extra `due to` that's concatenated with the strings that are passed in. It was accidentally added during a refactor, and I just removed it. If you look at the diff linked above, this is how it was previously: https://github.com/qbittorrent/qBittorrent/commit/fd89717330b826494ae37b48ddcdbada11c0b9a7#diff-53ed51e7986fe1b3e0bb6e6f53b67b6eL103.

Reply:
I think @glassez means to change this string to something like "%1 was blocked. Reason: %2."
and then remove the "due to" "because of" from the other strings. 
While you're at it, i would move the period to this string because it's common in all of them. Also i think "privileged ports" are not such a tech savvy term, therefore i would change "because it has a low port." to something like "it uses a privileged port", not to mention that "low port" is too vague.

Reply:
>The root of the problem was indeed this extra `due to` that's concatenated with the strings that are passed in.

This is only a symptom. The root of the problem is what I pointed out above (maybe it was really unclear). "Reason" string should contain only a description of the problem, without any "because of",  "due to", etc. The level at which the final formatting occurs can add this if required (or format the final string in some other way). The main rule is that lower layer shouldn't know how it is used by higher one. It should provide data in an independent/abstract form.

Reply:
@glassez @thalieht 

I understand now, thanks for the clarifications. Should I fixup the current commit with such a change, add a new separate commit, or submit another PR for this change?